user-timing/idlharness.any.js

```diff
@@ -0,0 +1,21 @@
+// META: global=window,worker
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+promise_test(async () => {
+  const idl_array = new IdlArray();
+  const idl = await fetch("/interfaces/user-timing.idl").then(r => r.text());
+  const hrtime = await fetch("/interfaces/hr-time.idl").then(r => r.text());
```

Comment:
you need to import /interfaces/performance-timeline.idl as well, since PerformanceMark depends on PerformanceEntry.

Reply:
Alright, I'll pull in https://github.com/web-platform-tests/wpt/pull/9809 to this PR.